autotst/calculator/statmech.py

```diff
@@ -402,7 +402,7 @@ def write_ts_input(self, transitionstate):
                    "spinMultiplicity = {}".format(
                        transitionstate.rmg_molecule.multiplicity),
                    "",
-                   "opticalIsomers = 1",
+                   "optical_isomers = 1",
```

Comment:
Arkane input should be camelCase? (I won't comment on any others - do a full search)